src/linear_solver_routines/block_col_projection.jl

```diff
@@ -0,0 +1,53 @@
+# This file is pat of RLinearAlgebra.jl
+"""
+    LinSysBlkColGent <: LinSysBlkColProjection
+
+A mutable structure that represents a standard block column projection method.
+
+# Aliases
+- `BlockCoordinateDescent`
+
+# Fields
+- `alpha::Float64`, a relaxation parameter that should be set between `0.0` and `2.0`
+- `gent::GentData`, a buffer to store important information related to the Gentleman's 
+incremental QR least squares solver.
+- `update::Union{Nothing, AbstractArray}`, a buffer for storing update.
+- `rowsize::Int64`, the maximum size of the Gentleman's row blocks. By defaul this is set to 10,000.
```

Comment:
- This needs to be clearer. What is the `maximum size of the Gentleman's row blocks` mean to your audience?
- Also, `defaul` ==> `default`.

For future consideration: Should we write `10000` rather than `10,000` or is `10,000` okay?

Reply:
I changed it to " the upper limit on the size of the row blocks in Gentleman's".

Reply:
In terms of `10,000` it may be better to write it as 10000 because of number writing conventions that vary across the world. It also may be better to write this in scientific notation.